<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QComboBox" name="comboBoxMatrix">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Нормированные значения</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Натуральные значения</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>220</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
